mediathread/settings_shared.py

```diff
@@ -40,6 +40,9 @@
     'mediathread.main.views.django_settings'
 )
 
+MIDDLEWARE = ['django_cookies_samesite.middleware.CookiesSameSite'] + \
```

Comment:
Just a quick question - does this middleware need to be first in the array? Could it be added to the array at Ln 46?

Reply:
Yes, it does need to be first (https://github.com/jotes/django-cookies-samesite#quickstart) - I first tested this by putting it in the array below, but then realized that's just appending to what's already in ccnmtlsettings, and that's why it wasn't working!

If you have an idea of a nicer way to prepend this to the array, that could work! I'll also add a comment here that this needs to be the first middleware.